rke/k8s_rke_system_images.go

```diff
@@ -6174,6 +6174,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.12
+		"v1.20.16-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.20.16-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
```

Comment:
rke-tools is bumped to v0.1.79, could we make sure this is the same as 1.20.16 in dev-v2.6

Reply:
Made the required change. Could you please review it again?